.gitmodules

```diff
@@ -1,3 +1,9 @@
 [submodule "myosuite/simhive/myo_sim"]
 	path = myosuite/simhive/myo_sim
 	url = https://github.com/MyoHub/myo_sim.git
+[submodule "myosuite/simhive/furniture_sim"]
```

Comment:
do we need the whole submodule? the repo is already quite big and furniture + obj will almost double its size